sysbuild/Kconfig.zip

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+menuconfig DFU_ZIP
```

Comment:
I've tried to run the mesh light sample with DFU with this option enabled in the sample sysbuild.conf file. But it seems no zip is generated at `light/build/dfu_application.zip` as expected. Later, I've also added `DFU_ZIP_BLUETOOTH_MESH_METADATA`. Then the zip file is generated, but it is missing the binary and manifest.

Is this going to be added later, or is it an oversight?